frog/imports/client/AccountModal/Login.js

```diff
@@ -20,17 +22,28 @@ const styles = (theme: Object) => ({
   },
   avatar: {
     margin: theme.spacing(1),
-    backgroundColor: theme.palette.secondary.main
+    backgroundColor: primaryColor
   },
   form: {
     width: '100%', // Fix IE 11 issue.
     marginTop: theme.spacing(1)
   },
   submit: {
-    margin: theme.spacing(2, 0, 2, 0)
+    background: primaryColor,
+    margin: theme.spacing(2, 0, 2, 0),
+    padding: theme.spacing(2),
+    boxShadow: '0 5px 20px rgba(0,0,0,0.05)',
+    transition: '.5s ease',
```

Comment:
Generally, we should aim for shorter transitions to make the interface feel more responsive